src/components/PageMetadata/Person.tsx

```diff
@@ -0,0 +1,40 @@
+import { Persona, PersonaSize } from '@fluentui/react/lib/Persona';
+import { TooltipHost, ITooltipHostStyles } from '@fluentui/react/lib/Tooltip';
+import { useId } from '@fluentui/react-hooks';
+import { IAuthorInfo } from './PageMetadata';
+
+interface IPersonProps {
+  author: IAuthorInfo
```

Comment:
Curiosity, if IPersonProps has only one property witch is author, why don't we just use IAuthorInfo and do not create a new interface?

Reply:
This will be easy to read. it's no effect with performance.

If I follow your suggestion, it will be like this

```ts
const Person = ({ author }:  {
  author: IAuthorInfo
}) => {
```

Reply:
Can we use Type alias like this?

```
type PersonProps = IAuthorInfo;
const Person = (person: PersonProps) => {

};
```
Optionally, you can destruction parameter as well.

Reply:
Nop, this will be error.

Reply:
@mildronize , please let me check there is no error in my editor, I need to run it.